js/render/draw_symbol.js

```diff
@@ -197,13 +203,15 @@ function drawSymbol(painter, layer, posMatrix, tile, bucket, bufferGroups, isTex
         gl.uniform1f(program.u_bearing, tr.bearing / 360 * 2 * Math.PI);
         gl.uniform1f(program.u_aspect_ratio, tr.width / tr.height);
 
+        painter.setDepthSublayer(1);
```

Comment:
Interesting, do you have a sense of why this is used in the GL Native implementation (https://github.com/mapbox/mapbox-gl-native/blob/master/src/mbgl/renderer/painter_symbol.cpp#L90-L111) ?